### PR TITLE
Add Japanese translation

### PR DESCRIPTION
**Pull request by tanakahisateru**
I finished Japanese translation.

I hope to have some discussion with our native language, though I don't agree some your statements but my opinion is a minor problem than that these statements are not shared by our local community.

### CODE REVIEW

File: sections/ja/welcome.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# 概要 #
+
+PHPプログラミングの世界では、ある一定のトレンドが「モダンPHP」として、一部の人によって (彼らの書籍やウェブサイト上で) 強烈に広められ、他のアプローチは、遅れていたり、愚かだったり、あるいは単に間違いだとみなされて、眉をひそめられています。
+
+それらの人々は、彼らの物事のやり方を他人に追従させようと、飽くなき努力を続けているように見えます。
+
+このウェブサイトは、PHP プログラミングに関する現実的な見解を示すために作られました。流行りのトレンド、理論、学問的な教示ではなく、経験と実践の帰着を書き記した視点です。
+
+ウェブサイト [PHP - The Wrong Way](http://www.phpthewrongway.com/) は生きたドキュメントですので、より多くの情報が明らかになにしたがって更新され続けます。
```
Comment (rono23) [inline sections/ja/welcome.md]: `なにしたがって` is typo?

Comment (tanakahisateru) [reply sections/ja/welcome.md]: Oh, soon

Comment (tanakahisateru) [reply sections/ja/welcome.md]: @rono23 fixed. thanks 👍

**Comment by unixdigest** [comment]
Fantastic work! Thanks a lot! 👍 

Ragarding having a discussion you're welcome to open an issue then we'll discuss there, but please see previous discussions (closed issues), maybe the topic has been covered :)

**Pull request merged**